docker/docker-compose.yml

```diff
@@ -41,6 +41,9 @@ services:
       - type: bind
```

Comment:
I had to replace `Config2.pm` with `Config2_pro.pm` at `line 39, 40` to avoid errors.

Reply:
I think it should probably be run in a 2nd backend container, so that both versions can be tested.

Reply:
Right, Config2.pm is to run an instance that looks like the public Open Food Facts platform, and Config2_pro.pm is to run an instance that looks like the producers platform on https://world.pro.openfoodfacts.org

There's probably a way to change the startup scripts to make it possible to specify which config file should be used.